### PR TITLE
Adding grouped layer control check boxes

### PR DESCRIPTION
**Pull request by corrigancd**
This PR adds checkboxes on the layer control groups:

AddingGroupedLayerControl

### CODE REVIEW

File: public/vis.less [hunk 1]
```diff
@@ -167,6 +174,16 @@
 
     .leaflet-control-layers-group {
       margin-bottom: .5em;
+
```
Comment (corrigancd) [inline public/vis.less]: This is the actual change

File: public/vis.less [hunk 2]
```diff
@@ -27,56 +27,63 @@
   }
 
   .siren-dashboard-drop-hover {
-    position: absolute;
+    position        : absolute;
```
Comment (makstakeda) [inline public/vis.less]: o____O

File: public/vis.less [hunk 3]
```diff
@@ -355,4 +372,4 @@
       padding-left: 2px;
     }
   }
-}
+}
```
Comment (makstakeda) [inline public/vis.less]: new line

Comment (Verdagio) [inline public/vis.less]: Missing newline?

File: public/vis.less [hunk 4]
```diff
@@ -247,14 +264,14 @@
 
   .tilemap {
     margin-bottom: 0px;
-    border: 0px;
+    border       : 0px;
```
Comment (Verdagio) [inline public/vis.less]: Your formatting seems a bit off here...

File: public/vis.less (hunk 1 above)
Comment (Verdagio) [reply public/vis.less]: what?

File: public/vislib/_map.js [hunk 5]
```diff
@@ -581,7 +581,7 @@ define(function (require) {
 
       this.saturateTiles(this._attr.isDesaturated);
 
-      this._layerControl = L.control.groupedLayers();
+      this._layerControl = L.control.groupedLayers(null, null, { groupCheckboxes: true });
```
Comment (Verdagio) [inline public/vislib/_map.js]: this looks fine

File: public/vis.less (hunk 2 above)
Comment (Verdagio) [reply public/vis.less]: How is this resolved... its the same as it was before

**Comment by makstakeda** [comment]
LGTM ^___^

**Comment by Verdagio** [comment]
LGTM

**Pull request merged**